cli/cmd/compliance_aws.go

```diff
@@ -285,6 +285,45 @@ To show recommendation details and affected resources for a recommendation id:
 		},
 	}
 
+	// complianceAwsEnableReportCmd represents the enable-report sub-command inside the aws command
+	// experimental feature
+	complianceAwsEnableReportCmd = &cobra.Command{
+		Use:     "enable-report <report_type>",
+		Aliases: []string{"enable"},
+		Hidden:  true,
+		PreRunE: func(_ *cobra.Command, args []string) error {
+			switch args[0] {
+			case "CIS":
+				args[0] = fmt.Sprintf("AWS_%s_S3", args[0])
+				return nil
+			case "AWS_CIS_S3":
```

Comment:
Is this strictly true? Should it be LW_AWS_S3 of AWS_LW_S3?

Reply:
Yeh, this is the enum value for the api report type so we allow it aswell as CIS

We will be allowing the value of with the version too, CIS_1_1